config/routes.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.routes.draw do
   resources :cards
-
-  # get 'home/index'
-  # get "home" => "home#index"
-  # get "/" => "home#index"
   root  'home#index'
+  match '/check',    to: 'home#check',    via: 'patch'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.